refs/schemas/priv/RFC-PRIV.md

```diff
@@ -0,0 +1,758 @@
+# Privacy Request Interchange Vocabulary (PRIV)
```

Comment:
This document is much more than a vocabulary. Should be renamed to `Language` or `Model`.

Reply:
Maybe better - exclude what is not vocabulary, into a spearate document so that we can still have PRIV (for privacy) in the name.

Reply:
Maybe better - exclude what is not vocabulary, into a spearate document so that we can still have PRIV (for privacy) in the name.